TODO/explain-activity-launch-mode-with-examples.md

```diff
@@ -24,29 +24,29 @@ Task #102 : affinity = “cn.six.adv”, size = 1
 
 — Activity One
 
-With this “adb shell dumpsys activity” command, we can explor the LaunchMode easier….
+拥有了这个神奇的命令——“adb shell dumpsys activity”，我们就可以更好地探索Activity的启动模式啦…
```

Comment:
Ok , we will chat in English from now on for helping each other improve our language.